packages/common/src/api/account.ts

```diff
@@ -37,9 +37,11 @@ const accountApi = createApi({
     resetPassword: {
       async fetch(args: ResetPasswordArgs, context) {
         const { email, password } = args
-        const { audiusBackend } = context
-
-        await audiusBackend.resetPassword(email, password)
+        const { userAuth } = context
+        await userAuth.resetPassword({
```

Comment:
This isn't the one we want to use. I realize I forgot to add the `authService` from common into AudiusQueryContext. And that's what we should be calling here (same as in the sagas)